.github/workflows/ci.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
+          - '1.3'
```

Comment:
I am wondering if maybe you want to just test Julia '1.3' (the lower bound) and '1' (the latest stable version)?

Reply:
no I decided against '1', because I find it confusing and not very helpful... The latest stable version doesn't change often enough, that its really worth having such a short cut ;)

Reply:
Fair enough :) But is it actually helpful to test '1.5'?

Reply:
Ah, if we test 1.3, probably not ;)